cell_classification/segmentation_data_prep.py

```diff
@@ -203,7 +220,11 @@ def make_tf_record(self, data_folders, tf_record_path):
         return None
 
     def calculate_normalization_matrix(
-        self, normalization_dict_path, normalization_quantile
+        self,
+        data_folders,
+        normalization_dict_path,
```

Comment:
I think it looks nicer to have the args to the function on a smaller number of lines


```
def calculate_normalization_matrix(self, data_folder,s normalization_dict_path, ... etc)
```

Reply:
These still need to be combined together into smaller number of lines